Resources/Prototypes/Entities/Clothing/Head/helmets.yml

```
@@ -29,15 +29,30 @@
 
 #Basic Helmet (Security Helmet)
 - type: entity
-  parent: [ClothingHeadHelmetArmoredBase, BaseSecurityContraband]
+  # Moffstation - Begin (Foldable Security Helmet)
+  parent: [ClothingHeadHelmetArmoredBase, BaseFoldable, BaseSecurityContraband]
   id: ClothingHeadHelmetBasic
-  name: helmet
+  name: security helmet
   description: Standard security gear. Protects the head from impacts.
   components:
+  - type: Foldable
+    canFoldInsideContainer: true
+    unfoldVerbText: fold-attach-visor-verb
+    foldVerbText: fold-detach-visor-verb
+  - type: FoldableClothing
+    foldedEquippedPrefix: visorless
+    foldedHeldPrefix: visorless
```

```suggestion
    foldedHeldPrefix: visorless
    foldedHideLayers:
    - HeadTop
    - HeadSide
    unfoldedHideLayers:
    - HeadTop
    - HeadSide
```
[These're the layers that're hidden by the base helme](https://github.com/moff-station/moff-station-14/blob/329fce495c0c1377dd469d1001b2d6f6a9e1dbef/Resources/Prototypes/Entities/Clothing/Head/helmets.yml#L12-L15)t, so I'm pretty sure the tests're complaining that these are being removed immediately when the entity is instantiated.